tests/sanity_helpers.py

```diff
@@ -31,10 +31,11 @@ def health_check(self, cluster_check=True, tries=20):
         logger.info("Checking cluster and Ceph health")
         node.wait_for_nodes_status()
 
-        ceph_health_check(namespace=config.ENV_DATA['cluster_namespace'], tries=tries)
         if cluster_check:
             self.ceph_cluster.cluster_health_check(timeout=60)
 
+        ceph_health_check(namespace=config.ENV_DATA['cluster_namespace'], tries=tries)
```

Comment:
Why are we making this change? 
In cluster health we are checking ceph health status, the test case will fail at this phase if it didn't reach to healthy state in 60sec(which is default) which is quite impossible in tier4 cases.
So, IMO better to not make any changes to this file as you are increasing the tries attempts to 60 in test case.

Please clarify it I am wrong here.

Reply:
if we do ceph_health_check first there is no use of  ceph_cluster.cluster_health_check () because of ceph_health_check passes then all pods will be in running state

Reply:
> if we do ceph_health_check first there is no use of ceph_cluster.cluster_health_check () because of ceph_health_check passes then all pods will be in running state. I don't see any issue here moving ceph_health_check() down

Not always. I think if some non ceph related pod is not in running state (like drain canary pod or collector pod) still the ceph health will shows as healthy :)
Just putting my point. 
If others are okay I am fine with this change.

Reply:
we'll need to run tier4 suites for verification with this change

Reply:
reverted back to default

Reply:
updated cluster_health_check to check pod status and expected count